Modify ``change`` field to be a json field. (#407)
jazzband/django-auditlog: CHANGELOG.md

# Changes

> #### Improvements

Reviewer: ```suggestion

#### Improvements
```

# Changes

> #### Improvements
> 
> - feat: Change `LogEntry.change` field to `JSONField` rather than `TextField` ([#407](https://github.com/jazzband/django-auditlog/pull/407))
> 

Reviewer: ```suggestion
#### Breaking Changes

- feat: Change `LogEntry.change` field type to `JSONField` rather than `TextField`. This change include a migration that may take time to run depending on the number of records on your `LogEntry` table ([#407](https://github.com/jazzband/django-auditlog/pull/407))

```

> - Updates `change` field on `LogEntry` model to be a json rather than a text field ([#407](https://github.com/jazzband/django-auditlog/pull/407))

Reviewer: ```suggestion
- feat: Change `LogEntry.change` field to `JSONField` rather than `TextField` ([#407](https://github.com/jazzband/django-auditlog/pull/407))
```
